hunts-api/routes/complete.js

```diff
@@ -0,0 +1,19 @@
+// complete.js
+
+const express = require('express');
+
+const router = express.Router();
+
+router.post('/', (req,res) => {
+    if((req.body.id == 'abcd-efgh-ijkl-mnop') && (req.body.image == 'base64String')){
+      res.status(200).send('SUCCESS');
```

Comment:
These responses should be json objects like mentioned in the acceptance criteria of the ticket:

```js
res.json({
  "status": "SUCCESS"
})
```